client/components/help-center/help-center-app.tsx

```diff
@@ -0,0 +1,31 @@
+import config from '@automattic/calypso-config';
+import { HelpCenter as HelpCenterStore } from '@automattic/data-stores';
+import HelpCenter from '@automattic/help-center';
+import { useDispatch } from '@wordpress/data';
+import React, { useCallback } from 'react';
+import './help-center-app.scss';
```

Comment:
Perhaps import `@automattic/calypso-color-schemes/root-only` directly?

Reply:
I tried but it failed 😞